changing text in program certificates

HamzaIbnFarooq opened the pull request:
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots

#### What are the relevant tickets?
https://github.com/mitodl/mitxpro/issues/2188

#### What's this PR do?
Changes content of program certificates

#### How should this be manually tested?
* Create a Program Certificate and match acceptance criteria of https://github.com/mitodl/mitxpro/issues/2188
* Make sure Course Certificates are not effected by this change.

#### Screenshots
**Program Certificate Web-View**:
![image](https://user-images.githubusercontent.com/42243411/115513956-0e650a00-a29d-11eb-8401-04729166a8ac.png)

**Program Certificate Printed-View:**
![image](https://user-images.githubusercontent.com/42243411/115514254-5a17b380-a29d-11eb-9bf7-dd3a21a729a7.png)

**Course Certificate Web-View**:
![image](https://user-images.githubusercontent.com/42243411/115514300-669c0c00-a29d-11eb-9882-608d2f3cd6e6.png)

**Course Certificate Printed-View:**
![image](https://user-images.githubusercontent.com/42243411/115514419-816e8080-a29d-11eb-9415-392bbc602ac5.png)

briangrossman commented:
@HamzaIbnFarooq The mockup looks good to me!

arslanashraf7 reviewed (approved):
Looks Good 👍 , It would be a good idea to add a screenshot of the print version and getting a green signal from Brian on that as well.

HamzaIbnFarooq commented:
Thanks @arslanashraf7 for pointing out that small difference, I have updated the PR, and here is the new screenshot of printed  certificate:
![image](https://user-images.githubusercontent.com/42243411/115204731-3f650380-a112-11eb-8cb7-c328d1deabdb.png)

cc: @briangrossman

arslanashraf7 commented:
@HamzaIbnFarooq, I tested it with the latest changes and they look good for program certificates but there seems to be a small problem with course certificate spacing, Adding the screenshot for that below, You can verify the actual look and feel for course certificates on RC.

Screenshot:
<img width="616" alt="Screenshot 2021-04-19 at 2 34 21 PM" src="https://user-images.githubusercontent.com/34372316/115215081-940d7c00-a11c-11eb-9ac0-b09ad51fef68.png">

HamzaIbnFarooq commented:
@arslanashraf7 I have updated the PR along with updating screenshots in description.

arslanashraf7 reviewed (approved):
Looks good 👍

briangrossman commented:
@HamzaIbnFarooq These look good overall. A couple small issues:

- The date is missing from your program certificates. The end date should be on the certificates (related issue: https://github.com/mitodl/mitxpro/issues/2176) 
- The spacing seems to be slightly different between the web and printed views of the course certificates. There's more space between the course title and the CEU's in the printed version. Please add the spacing to the web version so they look the same. 


**Web Version**
![image](https://user-images.githubusercontent.com/21342568/115401448-683fd400-a1b8-11eb-9eed-34ff534cfd73.png)

**Print Version (preferred)**
![image](https://user-images.githubusercontent.com/21342568/115401495-768df000-a1b8-11eb-8b82-8b68e90eda69.png)


Thank you!

HamzaIbnFarooq commented:
@briangrossman Thanks for the review.

> * The date is missing from your program certificates. The end date should be on the certificates (related issue: #2176)

It was a small devstack setup issue. updated the certificate screens now.

> * The spacing seems to be slightly different between the web and printed views of the course certificates. There's more space between the course title and the CEU's in the printed version. Please add the spacing to the web version so they look the same.

Updated, you may look into the attached screenshots now.

arslanashraf7 reviewed (approved):
The changes look fine.

To mention, my last comment was based on the difference in the look of the certificate between RC and this PR which I think was cleared out by Brian.

Feel free to merge after @briangrossman's green signal.

briangrossman commented:
@HamzaIbnFarooq 
The mockups look good. 

Looks good (اچھی) to me!

HamzaIbnFarooq commented:
@briangrossman شکریہ (Thanks)

briangrossman commented:
@HamzaIbnFarooq 
FYI: This is now on RC. I just tested it and it looks good.
